Twig/Extension/PostExtension.php

```diff
@@ -71,12 +71,13 @@ public function getFunctions()
 
     /**
      * @param int|Post $postId
+     * @param bool $isAbsolute
```

Comment:
Can you please update the PHPdoc in order to respect the checkstyle with the following:

```php
/**
  * @param int|Post $postId        A Wordpress post identifier
  * @param bool      $isAbsolute Determines if you want to retrieve ab absolute URL
  *
  * @return string
  *
  * @throws \UnexpectedValueException
  */
```

Thank you